Makefile

```
@@ -115,9 +115,12 @@ build/%.asc:
 
 test: test-unit ## Run the tests
 
-test-linter:  ## Run golangci-lint for the project
+test-linter:  ## Run golangci-lint with the enabled checks (used in the CI)
 	./hack/go-linter.sh
 
+test-dev-linter:  ## Run golangci-lint for with all checks (development purpose only)
```

I'd suggest making it a short, easy-to-type name and grouping it with the other developer related targets, like `tidy`, `format`, and `clean`
```suggestion
lint:  ## Run golangci-lint for with all checks (development purpose only)
```

---

Can be something as dev-lint?

---

But all tests are with test-<name>. In this case, it is a test and not a go dev command. So, see the other tests; test-unit, test-sanity, test-subcommand and then, would not make sense keep the same standard?

---

But your review as very valid, I did a few changes in order to make it more clear and follow up the current standard. WDYT? Is ok now?

---

> it is a test and not a go dev command.

We don't have a convention that says short, one-word make target names have to be `go` commands. I think if we have `make lint` devs will be more likely to see and use it, because it is much faster to recall and type (4 characters instead of 15).

Also, at this point it isn't really a test because we don't require that it passes. It is just informational.